README.md

```diff
@@ -205,7 +205,7 @@ Note: The `txt` field must be exactly 43 characters long, otherwise acme-dns wil
 
 4) Perform a DNS lookup to the test subdomain to confirm that everything is working properly:
 ```
-$ dig @ns.auth.example.org d420c923-bbd7-4056-ab64-c3ca54c9b3cf.auth.example.org
+$ dig @auth.example.org d420c923-bbd7-4056-ab64-c3ca54c9b3cf.auth.example.org
```

Comment:
actually the `@auth.example.org` should not be required at all. If the DNS setup is correct, this server should be queried eventually anyway.

Reply:
That's correct. And for testing purposes, even more usable actually. The current format is probably more usable for troubleshooting scenarios.

In general, I feel that the `README.md` should get restructured, and internally linked. For example, installation should link to the DNS configuration section, and testing it out to the (hopefully upcoming) troubleshooting section. Ideally these should follow each other in the document itself. I'll create an issue for that.